recipes/opensartoolkit/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "opensartoolkit" %}
+{% set version = "0.12.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opensartoolkit-{{ version }}.tar.gz
+  sha256: 867feb004333beb8245cbb6c548753bdc4d35545b99ee4496013a830442ea145
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.6
+    - setuptools
+    - setuptools
+  run:
+    - python >=3.6
+    - descartes
+    - fiona
+    - gdal >=2
+    - geopandas >=0.8
+    - godale
+    - imageio
+    - jupyterlab
+    - matplotlib-base
+    - numpy
+    - pandas
+    - psycopg2-binary
+    - pyproj >=2.1
+    - python
+    - rasterio
+    - requests
+    - retrying
+    - rtree
+    - scipy
+    - shapely
+    - tqdm
```

```suggestion
requirements:
  host:
    - pip
    - python >=3.6
    - gdal >=1.7
  run:
    - gdal >=1.7
    - python >=3.6
    - pyproj
    - descartes
    - godale
    - geopandas >=0.8
    - jupyterlab
    - psycopg2-binary
    - rasterio
    - requests
    - tqdm
    - imageio
    - rtree
    - retrying
```

Try this instead if you update to .15